mlflow/utils/_unity_catalog_utils.py

```diff
@@ -374,3 +390,55 @@ def get_full_name_from_sc(name, spark) -> str:
         return f"{catalog}.{name}"
     schema = spark.sql(_ACTIVE_SCHEMA_QUERY).collect()[0]["schema"]
     return f"{catalog}.{schema}.{name}"
+
+
+def is_databricks_sdk_models_artifact_repository_enabled(host_creds):
+    # Return early if the environment variable is set to use the SDK models artifact repository
+    if MLFLOW_USE_DATABRICKS_SDK_MODEL_ARTIFACTS_REPO_FOR_UC.defined:
+        return MLFLOW_USE_DATABRICKS_SDK_MODEL_ARTIFACTS_REPO_FOR_UC.get()
```

Comment:
```suggestion
        return True
```

Reply:
@harupy if user overrides this to be false explicitly, returning True is incorrect right?

Reply:
ah makes sense!